CustomLookup/CustomEntitySearch.cs

```diff
@@ -0,0 +1,187 @@
+using System;
```

Comment:
The repo organizes skills in Text/Geo/Vision folders. Looks like this one should be moved under Text.

Reply:
Moved to Text